site/src/helpers/styling.ts

```diff
@@ -0,0 +1,38 @@
+import { type Theme } from '@nivo/core';
+
+export const tooltipStyle: Theme = {
+  tooltip: {
+    container: {
+      background: 'rgba(0,0,0,.87)',
+      color: '#ffffff',
+      fontSize: '1.2rem',
+      outline: 'none',
+      margin: 0,
+      padding: '0.25em 0.5em',
+      borderRadius: '2px',
+    },
+  },
+};
+
+export const getTheme = (darkMode: boolean): Theme => {
```

Comment:
I think this should be called `getChartTheme` since it uses properties specific to the chart.